webui/react/src/components/UPlotChart.tsx

```diff
@@ -29,6 +29,8 @@ const UPlotChart: React.FC<Props> = ({ data, focusIndex, options }: Props) => {
   const chartDivRef = useRef<HTMLDivElement>(null);
   const scalesRef = useRef<Record<RecordKey, uPlot.Scale>>();
   const scalesZoomData = useRef<Record<string, ScaleZoomData>>({});
+  const isZoomed = useRef<boolean>(false);
+  const mousePosition = useRef<[number, number]>([ 0, 0 ]);
```

Comment:
**suggestion:** `mousePosition` should be able to be `undefined` by default. A default value of `[ 0, 0 ]` could potentially cause problems because it assumes the mouse always starts at that location.
```
const mousePosition = useRef<[ number, number ]>();
```